### PR TITLE
feat: changed dropzone

### PR DESCRIPTION
**Pull request by waynewee**
- changed graphics for dropzone 
- modularised dropzone logic into new DropZone Component
- added Card and ContentFrame to Revoke flow

### CODE REVIEW

File: src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx [hunk 1]
```diff
@@ -27,21 +28,13 @@ export const ConfigFileDropZone: FunctionComponent<ConfigFileDropZone> = ({ onCo
...
-          isDragActive ? "bg-gray-300" : "bg-white"
-        }`;
-
   return (
     <>
       <Title className="mb-8">Create Document</Title>
       <div {...getRootProps()}>
         <input data-testid="config-file-drop-zone" {...getInputProps()} />
-        <div className={dropZoneCSS}>
+        <DropZone isDragActive={isDragActive} error={Boolean(errorMessage || error)}>
```
Comment (simboonlong) [inline src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx:36]:
```suggestion
        <DropZone isDragActive={isDragActive} error={!!(errorMessage || error)}>
```

File: src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx [hunk 2]
```diff
@@ -59,7 +52,16 @@ export const ConfigFileDropZone: FunctionComponent<ConfigFileDropZone> = ({ onCo
           )}
           <div className="text-base text-gray-800 my-4">{errorMessage || error ? "Please try again." : "or"}</div>
           <Button className="bg-cerulean text-white hover:bg-cerulean-500 border-gray-300 px-12">Browse Files</Button>
-        </div>
+          <a
+            onClick={(e) => e.stopPropagation()}
```
Comment (simboonlong) [inline src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx:56]: can omit?

File: src/components/UI/DropZone/DropZone.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  error?: boolean;
+  children: React.ReactNode;
+}
+
+export const DropZone: FunctionComponent<DropZoneProps> = ({ children, isDragActive, error }) => {
+  let dropZoneCss = `cursor-pointer bg-white rounded-xl border-dashed border-2 items-center flex flex-col pt-16 pb-16 px-4 text-center`;
+
+  dropZoneCss += error ? ` border-red` : ` border-gray-300`;
+
+  dropZoneCss += error ? (isDragActive ? ` bg-gray-300` : `bg-red-100`) : isDragActive ? `bg-gray-300` : `bg-white`;
```
Comment (simboonlong) [inline src/components/UI/DropZone/DropZone.tsx:14]: um lemme think

File: src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx (hunk 2 above)
Comment (waynewee) [reply src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx:56]: cannot, cos the link is in the button i.e. clicking the link will cause redirect and cause the upload window to show

File: src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx (hunk 1 above)
Comment (waynewee) [reply src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx:36]: https://gist.github.com/arthurvi/66cb1e2bcfc92f99f465e0db04264367 
i like to use boolean cause more explicit

File: src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx (hunk 2 above)
Comment (simboonlong) [reply src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx:56]: ah okay got it

**Review by simboonlong — APPROVED** [review]

**Pull request merged**